pkg/system/publications/factory.go

```diff
@@ -29,7 +29,7 @@ func NewSystem(services system.Services) *System {
 	s := &System{
 		logger:    services.Logger.Named("publications"),
 		name:      services.Node.Name(),
-		announcer: services.Node,
+		announcer: node.NewReplaceAnnouncer(services.Node),
```

Comment:
Worth standardising on `announcer` vs `announcements` - I see both spellings used by different services. My vote goes for `announcer`

Reply:
Saw some `announce` fields too